### ISSUE TITLE
ValueError: time data does not match format '%m/%d/%Y'

### ISSUE THREAD

**Issue by jakegiganti** [arthurtyukayev/python-safer#12]
Calling `get_by_usdot_number` with 3279824 throws a value error. For some reason it's pulling the `Inspections/Crashes In US ` table instead of the `Safety Rating` table. It seems to only happen for random DOTs and I'm not sure where the mismatch is in the UI for this to happen. 

```
Traceback (most recent call last):
  File ".../CompanySnapshot.py", line 20, in <module>
    getCarrierInfoByDOT(3279824)
  File ".../CompanySnapshot.py", line 6, in getCarrierInfoByDOT
    company = client.get_by_usdot_number(dot)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File ".../venv/lib/python3.11/site-packages/safer/search.py", line 83, in get_by_usdot_number
    return Company(
           ^^^^^^^^
  File ".../venv/lib/python3.11/site-packages/safer/results.py", line 50, in __init__
    self.__safety_rating_date = datetime.strptime(data['safety_review_date'], "%m/%d/%Y") if data[
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '41' does not match format '%m/%d/%Y'
```

**Comment by arthurtyukayev** [comment]
Ref #13

**Comment by arthurtyukayev** [comment]
Should be fixed in latest v1.5
